cdc/sink/mq/codec/canal.go

```diff
@@ -81,7 +81,7 @@ func convertDdlEventType(e *model.DDLEvent) canal.EventType {
 	case mm.ActionAddColumn, mm.ActionDropColumn, mm.ActionModifyColumn, mm.ActionRebaseAutoID,
 		mm.ActionSetDefaultValue, mm.ActionModifyTableComment, mm.ActionRenameIndex, mm.ActionAddTablePartition,
 		mm.ActionDropTablePartition, mm.ActionModifyTableCharsetAndCollate, mm.ActionTruncateTablePartition,
-		mm.ActionAddColumns, mm.ActionDropColumns:
+		mm.ActionMultiSchemaChange:
```

Comment:
Why delete the `mm.ActionAddColumns, mm.ActionDropColumns:`? I think it's better to keep them for compatibility.

Reply:
These are designed for the old implementation of multi-schema change. They will be removed in v6.2.

Since we don't support using new version TiCDC on old version TiDB cluster, I think it should be OK?